migrationtools/databases.py

```diff
@@ -279,7 +279,7 @@ def update_identity(self, uid, identity_data):
         return self.execute(statement)
 
     def get_all_identities(self):
-        statement = select([self.identity]).order_by(self.identity.c.created_at)
+        statement = select([self.identity]).order_by(self.identity.c.created_at).limit(10)
```

Comment:
I'm a bit confused by the limit? Does this batch them, or does it only fetch 10?